src/CustomerBundle/Block/RecentCustomersBlockService.php

```diff
@@ -61,12 +64,12 @@ public function execute(BlockContextInterface $blockContext, Response $response
         ], $response);
     }
 
-    public function validateBlock(ErrorElement $errorElement, BlockInterface $block): void
+    public function validateBlock(ErrorElement $errorElement, BlockInterface $block)
```

Comment:
Why are you removing all these type declarations? :thinking:

Reply:
I dont want change parent declarations type when someone can extends this class. If class will be final then I will add declaration. (Final will be add in next major).

Reply:
But… you _are_ changing the declaration, so the extending classes have no choice but to have `: void` right now, don't they? The `: void` has been here for more than 2 years, it was introduced in 836b7eae7eaaec832cb5c7834723f1fb040b5a4f

Reply:
Look at: https://3v4l.org/bTGCg
I know it is not good change for code, but it eliminate posibillity of Fatal Error when extends this block

Reply:
>  but you cant remove it when parent have it.

Yeah so right now, every class that exists that extends this one has `: void` here. Please revert this change, it's only a loss of type information, and encourages people to not provide it when extending, which can only be worse (less information for SA tools)

Reply:
Ok. I add `NEXT_MAJOR: make this class final`.